src/core/config/transformer/preferred_name_setter.rs

```diff
@@ -0,0 +1,118 @@
+use std::collections::{HashMap, HashSet};
+
+use convert_case::{Case, Casing};
+
+use crate::core::config::Config;
+use crate::core::valid::Valid;
+use crate::core::Transform;
+
+// goes through operation type names and set's it's output type name from
+// suggested names;
+pub struct PreferredNameSetter(HashSet<String>);
```

Comment:
```rust
pub struct PreferredNameSetter(Vec<Suggestion>);

struct Suggestion {
    path: Path,
    type_name: String
}

struct Path {
    type_name: String,
    field_name: String,    
}
```

This is a special transformer that changes the name of a type present at a particular location.

Reply:
we can get same thing done without external transformer, we just have to assign higher priority to user suggested name.